subprojects/distributions/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -38,7 +38,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
     abstract String getDistributionLabel()
 
     int getLibJarsCount() {
-        162
+        163
```

Comment:
This number doesn't take into account transitive dependencies. It will have to be 169. You can verify the number running `./gradlew docs:docs distributions:allZip` the test `DistributionIntegrationSpec`.